demo/custom/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>custom-ui home</title>
+  <meta charset="UTF-8">
+  <script type="module" src="../../dist/mapml-viewer.js"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <script type="text/javascript" src="./custom-ui.js"></script>
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#">Custom User Interfaces</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+      aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+      <div class="navbar-nav">
+        <a class="nav-link active" href="#">Info <span class="sr-only">(current)</span></a>
+        <a class="nav-link" href="bootstrap-ui.html">Using Bootstrap</a>
+        <a class="nav-link" href="#">Using Custom CSS</a>
```

Comment:
```diff
- href="#"
+ href="custom-css.ui.html"
```

Also, don't you want the navbar on all `/custom/` pages so that you can navigate easily between them? If yes do you want me to make the necessary change or you got it?

Reply:
That was just in my repo from previous times, it's out of date but feel free to make any changes you feel are needed.